src/hooks/useKeyboardNavigation.js

```diff
@@ -71,6 +194,17 @@ const useKeyboardNavigation = ({
     if (nextSibling) nextSibling.focus();
   };
 
+  const activateCategoryByName = categoryName => {
```

Comment:
This goes against our react state that already does that. Setting the state is the preferred way.